authors/settings.py

```diff
@@ -138,6 +138,6 @@
     'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
+        # 'authors.apps.authentication.backends.JWTAuthentication',
```

Comment:
@danuluma why is this line commented out?

Reply:
Travis also throws that error.
![image](https://user-images.githubusercontent.com/43441107/50848228-bc766f80-1384-11e9-8c62-6aff7801c1fc.png)

Reply:
Okay you can leave it like that